keyboards/planck/rules.mk

```diff
@@ -62,7 +62,8 @@ AUDIO_ENABLE ?= no           # Audio output on port C6
 UNICODE_ENABLE ?= no         # Unicode
 BLUETOOTH_ENABLE ?= no       # Enable Bluetooth with the Adafruit EZ-Key HID
 RGBLIGHT_ENABLE ?= no        # Enable WS2812 RGB underlight.  Do not enable this with audio at the same time.
-API_SYSEX_ENABLE = yes
+API_SYSEX_ENABLE = no
```

Comment:
I'd like to keep this enabled by default for now - I have some things planned with the default layout for the Planck that will need this. We added warnings to the keymaps that were made too large by this inclusion.